pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -68,6 +68,7 @@ const TagNameSubnetPublicELB = "kubernetes.io/role/elb"
 // Currently we accept only the value "0.0.0.0/0" - other values are an error.
 // This lets us define more advanced semantics in future.
 const ServiceAnnotationLoadBalancerInternal = "service.beta.kubernetes.io/aws-load-balancer-internal"
+const ServiceAnnotationLoadBalancerName = "service.beta.kubernetes.io/aws-load-balancer-name"
```

Comment:
We also need a plan to move these out of "beta" and into "real".  @justinsb do you feel like you need beta for this?

`aws.service.kubernetes.io/load-balancer-name` or `service.aws.kubernetes.io/load-balancer-name` ?